wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -26,26 +53,153 @@ body.sensei {
 		display: none;
 	}
 
+	// Facilitator notes
+	#wporg-learn-facilitator-notes-toggle {
+		display: none;
+
+		&:checked + .wporg-learn-facilitator-notes-label + .wporg-learn-facilitator-notes-content {
+			display: block;
+			position: absolute;
+			top: calc(var(--sensei-lm-header-height));
+			right: 0;
+			width: 50%;
+			height: 100vh;
+			margin-top: 1px;
+			border-left: 1px solid var(--sensei-course-progress-bar-color);
+
+			.wporg-learn-facilitator-notes-close {
+				background: url(../../assets/icon-close.svg);
+				background-repeat: no-repeat;
+				background-position: center;
+				width: 24px;
+				height: 24px;
+				position: absolute;
+				right: var(--wp--preset--spacing--40);
+				top: calc(var(--wp--preset--spacing--30) + var(--wp--preset--spacing--20));
+				cursor: pointer;
+			}
+
+			@media (max-width: 782px) {
+				position: absolute;
+				top: var(--wp--preset--spacing--10);
+				left: 4%;
+				width: 92%;
+				height: 90vh;
+				border: 1px solid var(--sensei-course-progress-bar-color);
+				z-index: 5;
+				overflow: scroll;
+
+				+ .wporg-learn-facilitator-notes-backdrop {
+					content: "";
+					position: fixed;
+					top: 0;
+					left: 0;
+					width: 100%;
+					height: 100%;
+					background-color: var(--wp--preset--color--charcoal-5);
+					z-index: 1;
+				}
+			}
+		}
+	}
+
+	.wporg-learn-facilitator-notes-label {
+		cursor: pointer;
+		font-family: var(--wp--preset--font-family--inter);
+		font-size: var(--wp--preset--font-size--small);
+		color: var(--wp--custom--link--color--text);
+		text-decoration: underline;
+		text-underline-offset: 0.13em;
+		user-select: none;
+		line-height: 1.1875;
+
+		&:hover {
+			text-decoration: none;
+		}
+
+		@media (max-width: 782px) {
+			display: none;
+		}
+	}
+
+	.wporg-learn-facilitator-notes-content {
```

Comment:
I notice that the modal content overflow is hidden. Some of the Lesson Plan content is very long so this will need to be scrollable.

Reply:
Thanks. c125bcb